src/sass/components/_mention.scss

```diff
@@ -1,28 +1,28 @@
 .mention {
+    border-radius: 5px;
     padding: 0 5px 0 20px;
-    -webkit-border-radius:5px;
-    -moz-border-radius:5px;
-    border-radius:5px;
     line-height: 10pt !important;
+    -moz-border-radius: 5px;
+    -webkit-border-radius: 5px;
 
-    &.me,&.all {
+    &.me, &.all {
         background-color: #8b8b8b;
-        color: white;
+        color: #fff;
```

Comment:
Why does the linter not complain about non-variable colors here?

Reply:
It does. I just stopped fixing them because I saw discussion potential. :wink: